docs/reference/search/search-your-data/learning-to-rank-model-training.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[learning-to-rank-model-training]]
+=== Deploy and manage Learning To Rank models
+++++
+<titleabbrev>Deploy and manage LTR models</titleabbrev>
+++++
+
+preview::["The Learning To Rank feature is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but this feature is not subject to the support SLA of official GA features."]
+
+NOTE: This feature is available for Elastic Stack versions 8.12.0 and newer and requires a https://www.elastic.co/pricing[Platinum subscription] or higher.
+
+[discrete]
+[[learning-to-rank-model-training-workflow]]
+==== Train and deploy a model using Eland
+
+https://xgboost.readthedocs.io/en/stable/[XGBoost^] model training typically leverages a standard, Python data science toolkit such as Pandas and scikit-learn
+
+We have developed an example notebook available https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/08-learning-to-rank.ipynb[here^], detailing an end-to-end model training and deployment workflow.
+
+We highly recommend integrating https://eland.readthedocs.io/[eland^] into your workflow since it provides some important features in the workflow to integrate Learning To Rank in {es}:
```

Comment:
```suggestion
We highly recommend using https://eland.readthedocs.io/[eland^] in your workflow, because it provides important functionalities for working with LTR in {es}. Use eland to:
```

Reply:
eland or Eland? Just a question of consistency...